system/api-server/src/file-upload/file-upload.service.ts

```diff
@@ -33,7 +33,7 @@ export class ImageUploadService {
       Bucket: AWS_S3_BUCKET_NAME,
       Body: imageBuffer,
       Key: fileName,
-      ACL: 'public-read',
+      ACL: 'public-read', 
```

Comment:
Public read seems pretty permissive - are there other permissions you would like to explore here and how would you think about implementing it?

Reply:
I could make it private and use a signed URL to access the files. I haven't done this before though.